content/docs/2.1/operate/cluster.md

```diff
@@ -12,7 +12,7 @@ Here is an overview of all KEDA deployments and the supported replicas:
 
 | Deployment     | Support Replicas        | Reasoning                     |
 |----------------|-------------------------|-------------------------------|
-| Operator       | 1                       |                               |
+| Operator       | 2                       |                               |
```

Comment:
```suggestion
| Operator       | 1                       | While you can run more replicas of our operator, only one operator instance will be the leader and serving traffic. You can run multiple replicas, but they will not improve the performance of KEDA. This behavior is how operators work. |
```

Reply:
What about, or something like this more polished:
```
While you can run more replicas of our operator, only one operator instance will be the leader and serving traffic. You can run multiple replicas, but they will not improve the performance of KEDA, it could only reduce a downtime during a failover.
```

Reply:
Even better!